docs/metadata-ui.rst

```diff
@@ -0,0 +1,190 @@
+
+- How to :ref:`create or adjust a local CSV file containing metadata<Populatemetadatacsv>` for creation of OMERO.tables containing numbers and text. 
+
+Resources
+---------
+
+-  Data: example images from
+
+   -  IDR data `idr0021-lawo-pericentriolarmaterial/experimentA <https://idr.openmicroscopy.org/webclient/?show=project-51>`_
+
+-  Annotation CSV files define the content of OMERO.tables for each image or each well.
+
+   - :download:`four-images.csv <../scripts/four-images.csv>`
+   - :download:`simple-screen.csv <../scripts/four-images.csv>`
+
+-  ``omero-metadata`` plugin (necessary for having full set of features in ``Populate Metadata`` script).
+
+   - https://pypi.org/project/omero-metadata/
+
+Setup
+-----
+
+**omero-metadata plugin installation**
+
+.. note::
+      For the best experience, the ``omero-metadata`` plugin should be installed on your OMERO.server. The ``Populate Metadata`` script tries to reuse the code of the ``omero-metadata`` plugin. If the plugin is not found on the server, the ``Populate Metadata`` script falls back on a deprecated code, with limited set of features. The ``omero-metadata`` plugin installation is typically done by the administrator of the OMERO.server.
+
+- In your OMERO.server environment, go to the environment where you installed your OMERO.cli as specified under - `CLI installation <https://docs.openmicroscopy.org/omero/latest/users/cli/installation.html>`_.
+
+- Activate the virtual environment.
+
+- Run::
+    
+    $ pip install omero-metadata
+
+**Populate Metadata script**
+
+No explicit installation necessary, shipped with the OMERO.server.
+
+
+Step-by-step
+------------
+
+.. _Populatemetadatapdi:
+
+*Project/Dataset/Image*
+~~~~~~~~~~~~~~~~~~~~~~~
+
+#.  Log in to OMERO.web, create a new Dataset and copy into it four images, preferably images which have no OMERO.tables on them. Note the name of the images you are copying in.
+
+    |image4|
+
+#.  Download :download:`four-images.csv <../scripts/four-images.csv>`. Open the CSV file in Excel and edit the name of the images in the first column to match the names of the images you copied into your Dataset in the previous step. Also, edit the name of the Dataset in the second column to match the name of your Dataset in OMERO.web. Save the file locally as CSV.
+
+#.  (Optional) In your OMERO.web, upload the CSV file you just saved and attach it onto the Dataset you created previously. Alternatively, you can skip this step, and point the ``Populate Metadata`` script to the local CSV, as explained below.
+
+    |image5|
+
+#.  Select the Dataset you created and attached to it the CSV. Find the script icon |image6| above the central pane, expand it and find the ``Import scripts`` section. In there, select the ``Populate metadata`` script.
+
+    |image7|
+
+#.  If you did not attach the CSV to the Dataset, you can now click on the ``Browse`` button and select the CSV from your local machine.
+
+#.  Run the script.
```

Comment:
"Click OK to run the script, and wait for it to show as complete in the 'Activities' panel". (or something like that)

Reply:
fixed in https://github.com/ome/omero-guide-upload/pull/13/commits/f7643b46e97c45e7a8200b999431385420162be5